ntbs-service/Pages/Notifications/Edit/PatientDetails.cshtml.cs

```diff
@@ -197,7 +197,7 @@ public async Task<ContentResult> OnPostValidatePatientDetailsDate([FromBody]Date
         public async Task<JsonResult> OnPostNhsNumberDuplicates([FromBody]NhsNumberValidationModel validationData)
         {
             var group = await NotificationRepository.GetNotificationGroupAsync(validationData.NotificationId);
-            return new JsonResult(await GenerateDuplicateNhsNumberNotificationUrlsAsync(validationData.NhsNumber, group));
+            return new JsonResult(await GenerateDuplicateNhsNumberNotificationUrlsAsync(NotificationFieldFormattingHelper.FormatNhsNumberForModel(validationData.NhsNumber), group));
```

Comment:
This only catches duplicates in the warning message in the front end... I assume this means that the code was already properly checking for duplicates when the notification is actually saved?